front/src/reducers/osrdconf/types.ts

```diff
@@ -26,7 +26,9 @@ export interface OsrdConfState extends InfraState {
   scenarioID?: number;
   pathfindingID?: number;
   timetableID?: number;
-  electricalProfileSetId?: number;
+  electricalProfileSetID?: number;
+  workScheduleGroupID?: number;
+  searchDatetimeWindow?: [Date, Date];
```

Comment:
nit: I prefer to have an object with begin and end (or low and high), I think it's more explicit when we use it afterwards
```suggestion
  searchDatetimeWindow?: { begin: Date, end: Date };
```